crates/host/src/secrets.rs

```diff
@@ -26,6 +26,13 @@ pub(crate) struct SecretReference {
     /// The version of the secret to retrieve. If not supplied, the latest version will be used.
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub version: Option<String>,
+    /// The policy that defines configuration options for the backend.
+    pub policy_properties: String,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+struct SecretReferenceProperties {
+    properties: HashMap<String, String>,
```

Comment:
Do we need this struct anymore? Since we're just passing the policy_properties through

Reply:
Nope, left over from what I was doing before.